src/hermes.jsx

```diff
@@ -0,0 +1,338 @@
+'use strict';
+import React, {PropTypes} from 'react';
```

Comment:
I have stopped using the `class MyComponent extends Component` way, because it forces you to jump through hoops to get the most important part, `propTypes`, at the top of the file.  "Using es2015" does not mean you have to use every feature all the time, it is up to you to choose the features when they make the most sense.